components/date-field/src/test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { shallow, mount } from 'enzyme';
+import { shallow, mount, render } from 'enzyme';
+import sinon from 'sinon';
```

Comment:
Not sure Sinon is required here since Jest has mock functions.

Reply:
How would you change it?

Reply:
I'd replace with `sinon.spy()` `jest.fn()`. [Here's](https://github.com/stevesims/govuk-frederic/blob/master/components/compact-table-accordion-group/src/test.js) an example of some Jest mocks/spies being used. :)

Reply:
Awesome, is that something you'd like to do? I think Alasdair is pretty stacked - I guess we can ask him also if there is any reason he used sinon here.

Reply:
I could do with changing my work up a little bit so I'll do this now : )

Reply:
Cool :) I'm occupied trying to get IAM working in DevSt2 at the moment.

Reply:
No worries, took a sec, sorted : )